js/control/Export.js

```diff
@@ -23,7 +23,9 @@ BR.Export = L.Class.extend({
 
     _export: function() {
         var exportForm = document.forms['export'];
-        var format = exportForm['format'].value;
+        var format =
+            exportForm['format'].value ||
+            $('#export-format input:radio:checked');
```

Comment:
This is only returning the jQuery object, not the value - I think `.val()` needs to be added:

```
> $('#export-format input:radio:checked')
jQuery.fn.init [input#format-gpx.form-check-input, prevObject: jQuery.fn.init(1)]
> $('#export-format input:radio:checked').val()
"gpx"
```

When commenting out line 27 (testing with Chrome), the download URL is fixed because `format != null`, but the file extension is: `[object Object]`.

Reply:
Damn, you're right I forgot to add it. I tested it via the dev console and then did some refactoring but then forgot it... *sight*. Thanks, as always.